File: TaskBookAPI/TaskBookAPI/Models/DB/Book.cs
namespace TaskBookAPI.Models.DB
{
    public partial class Book
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public int Year { get; set; }
        public int Authorid { get; set; }
        public int Genreid { get; set; }

        public virtual Author Author { get; set; } = null!;
        public virtual Genre Genre { get; set; } = null!;
    }
}

File: TaskBook/src/models/book.js
export default class Book {
    constructor(id,name, author, year, genre, authorId, genreId){
        this.id = id
        this.name = name
        this.author = author
        this.year = year
        this.genre = genre
        this.authorId = authorId
        this.genreId = genreId
        this.selected = false
    }
}